server/index.js

```diff
@@ -38,7 +38,6 @@ app.post("/config", (req, res) => {
     corpus_id,
     customer_id,
     api_key,
-    hf_token,
```

Comment:
hf_token is used for the old code (pre-FCS) to compute this directly from Huggingface